test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +42,37 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
```

Comment:
I'm not sure what code style is here, but usually, methods order in .h and .cxx files should match.  In many code style guides first, methods are ordered by access specifiers public then protected and the last one is private. And within those blocks, static methods go in front and then normal methods.

Reply:
My bad, the order is correct. I didn't notice the constructor and destructor are deleted and the default constructor is private. All good.